src/router/RouterOptions.php

```diff
@@ -0,0 +1,16 @@
+<?hh // strict
+
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HackRouter;
+
+type RouterOptions = shape(
```

Comment:
- Should probably be called `BaseRouterOptions` as it can't be used by any other router if that router itself has more options; making the shape allow subtypes would probably be a bad change for killing typo detection.
- use_get_responder_for_head would be clearer

Reply:
> making the shape allow subtypes would probably be a bad change for killing typo detection.

hm ? extra fields are not allowed here

Reply:
Yeah, I was mentioning that as a bad alternative to renaming it :)

Reply:
resolved then :)